python/demo/demo-part1.ipynb

```diff
@@ -54,9 +52,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "d[Coord.Row] = ([Dim.Row], np.arange(3))\n",
-    "d[Data.Value, \"alice\"] = ([Dim.Row], [1.0,1.1,1.2])\n",
-    "d[Data.Variance, \"alice\"] = ([Dim.Row], [0.01,0.01,0.02])\n",
+    "d.set_coord(sc.Dim.Row, sc.Variable([sc.Dim.Row], values=np.arange(3)))\n",
+    "d[\"alice\"] = sc.Variable([sc.Dim.Row], values=[1.0,1.1,1.2], variances=[0.01,0.01,0.02], unit=sc.units.m)\n",
     "d"
```

Comment:
I guess with the additions coming giving scipp the ability to render itself, we might also use that in this notebook in particular.

Reply:
I didn't think that was fully operational yet

Reply:
As far as I know it's not. I was just stating that this might be a good place to exercise it. Wouldn't form part of this PR.

Reply:
I added a cell to showcase this now.